docs/usage.md

```diff
@@ -30,18 +30,18 @@ SAMPLE1,path/to/SAMPLE1.bam,female
 SAMPLE2,path/to/SAMPLE2.bam,male
 ```
 
-| Column    | Description|
+| Column    | Description |
 | --------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| `sample`  | Custom sample name. This entry will be identical for multiple sequencing libraries/runs from the same sample. Spaces in sample names are automatically converted to underscores (`_`). |
-| `fastq_1` | Full path to FastQ file for Illumina short reads 1.                                                         |
-| `fastq_2` | Full path to FastQ file for Illumina short reads 2.     
-| `bam` | Full path to BAM file.                                                          |
+| `sample`  | Custom sample name. This entry will be identical for multiple sequencing libraries/runs from the same sample. Spaces in sample names are automatically converted to underscores (`_`).                                                               |
+| `fastq_1` | Full path to FastQ file for Illumina short reads 1.                                           |
```

Comment:
This is not inline with line 34. All `|` should end in the same column as line 34, like

```
| aaaa      | bbbb |
|---------- | -----|
```